util/storage-samples/storage-owner-sample/src/main/java/org/fidoalliance/fdo/storage/OwnerDbManager.java

```diff
@@ -557,7 +557,9 @@ public Composite getSystemResources(DataSource ds,
         + "(ARCHITECTURE_TAG = ? OR ARCHITECTURE_TAG IS NULL ) AND "
         + "(CONTENT_TYPE_TAG = 'fdo_sys:filedesc' "
         + "OR CONTENT_TYPE_TAG = 'fdo_sys:exec' "
-        + "OR CONTENT_TYPE_TAG = 'fdo_sys:active') "
+        + "OR CONTENT_TYPE_TAG = 'fdo_sys:active' "
```

Comment:
Document additional fdo_sys keys

2 more keys have been added to fdo_sys module (ismore and isdone). These need to be added to github.io documentation for fdo_sys module.